nifty-renderer-libgdx/src/main/java/de/lessvoid/nifty/gdx/render/GdxGL.java

```diff
@@ -1,10 +1,11 @@
 package de.lessvoid.nifty.gdx.render;
 
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.graphics.GL10;
-import com.badlogic.gdx.graphics.GL11;
 
+import com.badlogic.gdx.utils.BufferUtils;
 import de.lessvoid.nifty.render.batch.spi.GL;
+import org.lwjgl.opengl.GL11;
```

Comment:
Please replace all static references to LWJGL namespace with the constants/methods exposed by `com.badlogic.gdx.graphics` namespace instead.

So constants should reference the public static members of `com.badlogic.gdx.graphics.GL20` (or GL30) and methods should all use `Gdx.gl.[methodName]`

Other than that, it looks good. Thanks for the contribution!